audio-worklet/noisegate-test.html

```diff
@@ -253,15 +254,15 @@
         const numberOfChannels = 2;
 
         // The beneathThresholdGain and linearThreshold are set so that
-        // (1 * sqrt(2) + beneathThresholdGain^2 * sqrt(2)) / 2
-        // < linearThreshold since the envelope will scale up values by the
-        // square root of two and the test is designed so that signals after
-        // timeToThreshold are muted in both channels.
+        // (1 * 2 + beneathThresholdGain * 2) / 2 < linearThreshold since the
```

Comment:
Any reason not to write this formula as `1 + beneathThresholdGain < linearThreshold < linearThreshold`?